experiments/functional/backup_and_restore/setup_dependency.yml

```diff
@@ -194,6 +201,20 @@
 
 - block:
 
+    - name: Replacing profile spec in volumesnapshotlocation.yml
+      replace:
+        path: ./minio-volume-snapshot-location.yml
+        regexp: "#profile: new-profile"
+        replace: "profile: {{ profile_name }}"
+      when: lookup('env','PROFILE') != "default"
+
+    - name: Replacing profile spec in backupstoragelocation.yml
```

Comment:
This task is updating for VSL instead of BSL. Please check my other comment about mentioning profile during installation itself.

Reply:
I made the change to perform replacement on correct file.